docs/src/index.rst

```diff
@@ -12,106 +12,13 @@
 Welcome
 #######
 
-Welcome to the |cdk-long| (|cdk|) User Guide!
+Welcome to the |cdk-long| (|cdk|)!
 
 The |cdk| is an infrastructure modeling framework that allows you to define your
 cloud resources using an imperative programming interface. *The CDK is currently
 in developer preview. We look forward to community feedback and collaboration*.
 
-To get started see :doc:`getting-started`
-
-.. image:: screencast.gif
-
```

Comment:
Is this commented out?

Reply:
Yes, it's in the guide now: https://docs.aws.amazon.com/CDK/latest/userguide/what-is.html